exec/java-exec/src/main/java/org/apache/drill/exec/client/DrillClient.java

```diff
@@ -223,19 +223,65 @@ public void connect(Properties props) throws RpcException {
     connect(null, props);
   }
 
+  /**
+   * Function to populate the endpointList with information of all the drillbits
```

Comment:
Thanks for the Javadoc!

Function to populate -> Populate

information of all the -> the list of